ultra/ultra/train.py

```diff
@@ -94,6 +147,52 @@ def train(
         for agent_id, agent_spec in agent_specs.items()
     }
 
+    # Define an 'etag' for this experiment's data directory based off policy_classes.
+    # E.g. From a ["ultra.baselines.dqn:dqn-v0", "ultra.baselines.ppo:ppo-v0"]
+    # policy_classes list, transform it to an etag of "dqn-v0:ppo-v0".
+    etag = ":".join([policy_class.split(":")[-1] for policy_class in policy_classes])
+
+    return agent_ids, agent_classes, agent_specs, agents, etag
+
+
+def _save_agent_metadata(experiment_dir, agent_ids, agent_classes, agent_specs):
```

Comment:
I'm thinking maybe we should pass in the desired filename to this function too (see original comment on this function), so that we keep the filename in the calling function, rather than have the filename be part of the calling and called function. It may just be cleaner and easier if the string literal ("agent_metadata.pkl") is kept in one place. Let me know your thoughts.

Reply:
This will also generalize the filename itself, thus no hard coding would be require. That makes sense. Let me know the changes in 450a91b

Reply:
Looks good, but I don't think we use `_AGENT_METADATA_FILENAME` anywhere. Let's just get rid of it.